### PR TITLE
add share on reddit button

### LINKED ISSUE #2753: Community - Add Reddit share button
A lot of users like to share their Steemit posts on Reddit to attract attention from users outside the platform. There is a request to add a Reddit share button to the share links.

### PR DESCRIPTION
**Pull request by roadscape**
Replaces #2757 by @netuoso:
> Pretty basic. Adds the Reddit SVG icon (greyscale) and appends it to the share_menu on PostFull.jsx.
> 
> Due to weird behavior when opening a new window for the Reddit share, the button replaces the current window.location with the required link for the Reddit share for enhanced UX. Up to suggestions on getting the Reddit share window to be properly opened and sized in a new popup window.
> 
> closes #2753 
> 
> screen shot 2018-04-23 at 6 41 56 pm
> screen shot 2018-04-23 at 6 42 08 pm

### CODE REVIEW

**Comment by roadscape** [comment]
image

File: src/app/components/cards/PostFull.jsx [hunk 1]
```diff
@@ -170,6 +171,18 @@ class PostFull extends React.Component {
...
+    redditShare(e) {
+        serverApiRecordEvent('RedditShare', this.share_params.link);
+        e.preventDefault();
+        const s = this.share_params;
+        const q =
+            'title=' +
+            encodeURIComponent(s.title) +
+            '&url=' +
+            encodeURIComponent(s.url);
+        window.location = '//www.reddit.com/submit?' + q;
```
Comment (gl2748) [inline src/app/components/cards/PostFull.jsx]: Don't we what `https://` here?

Comment (gl2748) [reply src/app/components/cards/PostFull.jsx]: Also, the other share options are using `window.open`, perhaps that's preferable.

Comment (roadscape) [reply src/app/components/cards/PostFull.jsx]: Good to be explicit, though should use `https` since steemit.com will always be

**Pull request merged**